services/service_instance_lifecycle.go

```diff
@@ -288,13 +279,13 @@ var _ = ServicesDescribe("Service Instance Lifecycle", func() {
 					Expect(bindService).To(Exit(0), "failed binding app to service")
 				})
 
-				Context("when there is an existing binding", func() {
+				FContext("when there is an existing binding", func() {
```

Comment:
I don't think you meant to leave this focus (and the one below) in

Reply:
Thanks for catching it. I removed it now